models/_generate_layers.py

```diff
@@ -246,7 +247,7 @@ def module_call_needs_suppress_shadow_builtin_warning(self):
         return True
     return False
 
-  _IgnoreParamNames = {
+  IgnoreParamNames = {
```

Comment:
This should not be needed when `get_super_call_kwargs_repr` is a member function here.